### PR TITLE
Update issue templates with notices about the version and forks of paper.

### PR DESCRIPTION
**Pull request by aurorasmiles**
Add comment about the versions and forks

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/behavior-bug---plugin-incompatibility.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ about: Server Bug or Plugin Incompatibility
 
 ---
 
+<!--- Before reporting an issue here, please make sure you are on the latest version of Paper.  -->
+<!--- Forks of Paper receive no support here. Please make sure that your issue is reproducable on Paper first.  -->
```
Comment (Proximyst) [inline .github/ISSUE_TEMPLATE/behavior-bug---plugin-incompatibility.md]: Only two hyphens for comments. Also, a new comment directive here when theres another right above seems wasteful.

**Review by Proximyst — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/behavior-bug---plugin-incompatibility.md (hunk 1 above)
Comment (aurorasmiles) [reply .github/ISSUE_TEMPLATE/behavior-bug---plugin-incompatibility.md]: I think it looks prettier with a new comment, but I don't think that matters too much

Comment (Proximyst) [reply .github/ISSUE_TEMPLATE/behavior-bug---plugin-incompatibility.md]: In that case, at least remember comments are *two* hyphens, not three.

Comment (aurorasmiles) [reply .github/ISSUE_TEMPLATE/behavior-bug---plugin-incompatibility.md]: I already changed that :D

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ about: Suggest an idea for this project
 
 ---
 
+<!-- Before requesting a new feature, please make sure you are using the latest version and that the feature you are requesting is not already in Paper -->
```
Comment (Proximyst) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Full stop :>

**Review by Proximyst — DISMISSED** [review]

**Review by Proximyst — DISMISSED** [review]

**Comment by Chew** [comment]
Should be a general thing about "Paper only for supported versions", because spigot and craftbukkit aren't forks but are also not supported.

**Review by Proximyst — APPROVED** [review]
Sorry for making this my testing grounds <3

**Review by DenWav — APPROVED** [review]

**Pull request merged**